frontend/src/views/camp/navigation/CampNavigation.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <NavDesktop v-if="$vuetify.breakpoint.smAndUp" :camp="camp" />
+  <div v-else>
+    <NavMobileBottom :camp="camp" @input="open = $event" />
+    <NavMobileSidebar :camp="camp" v-model="open" />
+  </div>
+</template>
+
+<script>
+import NavDesktop from '@/views/camp/navigation/NavDesktop.vue'
+import NavMobileBottom from '@/views/camp/navigation/mobile/NavMobileBottom.vue'
+import NavMobileSidebar from '@/views/camp/navigation/mobile/NavMobileSidebar.vue'
+
+export default {
+  name: 'CampNavigation',
```

Comment:
Usually the more general term should come before the more specific one in component names, see https://v2.vuejs.org/v2/style-guide/?redirect=true#Order-of-words-in-component-names-strongly-recommended. That is why we previously had a component named `NavigationCamp`. Are you sure this is the best name for this component?